prisma/schema.prisma

```diff
@@ -15,24 +15,12 @@ datasource db {
 }
 
 model User {
-  id String @id @default(uuid())
-
-  email    String?   @unique
-  name     String
-  password String
-  posts    Post[]
-  sessions Session[]
-
-  createdAt DateTime @default(now())
-  updatedAt DateTime @default(now()) @updatedAt
-}
-
-model Session {
-  id String @id @default(uuid())
+  id      String @id @default(uuid())
+  clerkId String
```

Comment:
`@unique`

Reply:
I didn't put it to allow multiple users (on different tenant) to have the same clerkId. Like you could be managing multiple orgs and in the db it will correspond to two different users with the same clerk id

Reply:
when we have a tenant table, I could add `unique('clerkId', 'tenantId')`

Reply:
Yeah, I guess tenants have 'their own' users, but this is the id coming from clerk and shouldn't ever be duplicated, unless we make the tenants bring their own clerk account like with twilio?